tests/Number/RealTest.php

```diff
@@ -63,12 +63,35 @@ public function testIfRealIsIntegerOrNotShouldSuccess()
         $this->assertTrue($i->isInteger());
     }
 
-
     public function testStringContextShouldSuccess()
     {
         $i = new Real(2.56);
         $d = new Real(0.56);
         $this->assertEquals('2.56', "$i");
         $this->assertEquals('0.56', "$d");
     }
+
+    public function testGetShouldBePI()
+    {
+        $real = new Real(2.56);
+        $this->assertEquals(M_PI, $real->__get('pi')->__toString());
+    }
+
+    public function testGetShouldBeE()
+    {
+        $real = new Real(2.56);
+        $this->assertEquals(M_E, $real->__get('e')->__toString());
```

Comment:
Same idea as in https://github.com/malenkiki/math/pull/25/commits/fa42d3500c88374787a532d35f0fcabf8242250e#r150423847